scripts/run-dev-win.sh

```
@@ -0,0 +1,93 @@
+  export CEF_RUNTIME_PATH
+fi
+
+PATH_PREFIX="/c/Program Files/CMake/bin:$(dirname "$NINJA_EXE")"
+if [[ -n "${CEF_RUNTIME_PATH:-}" ]]; then
+  PATH_PREFIX="$PATH_PREFIX:$CEF_RUNTIME_PATH"
+fi
+export PATH="$PATH_PREFIX:$PATH"
+
+"$PNPM_EXE" tauri:ensure
+"$PNPM_EXE" core:stage
+source ../scripts/load-dotenv.sh
```

Shellcheck (0.11.0)

[info] Not following: ../scripts/load-dotenv.sh was not specified as input (see shellcheck -x).

(SC1091)